rpi-source

```diff
@@ -408,8 +438,13 @@ if args.default_config or not kernel.config:
     info(".config (generating default)")
     if processor_type == 0:
         sh("cd %s && make bcmrpi_defconfig" % (linux_symlink,))
-    elif processor_type == 1 or processor_type == 2:
+    elif processor_type == 1:
         sh("cd %s && make bcm2709_defconfig" % (linux_symlink,))
+    elif processor_type == 2:
+        if architecture_type == 0:
+            sh("cd %s && make bcm2709_defconfig" % (linux_symlink,))
+        elif architecture_type == 1:
+            sh("cd %s && make bcmrpi3_defconfig" % (linux_symlink,))
```

Comment:
BCM2710 should also be using `bcm2711_defconfig` - the `bcmrpi3_defconfig` is a misleading unofficial defconfig that may be deleted.

Reply:
Resolved in [2a5ba55](https://github.com/RPi-Distro/rpi-source/pull/29/commits/2a5ba55873bed11c6c5448c9ca7c171f18259abf)